event_attribution_reporting_clicks.md

```diff
@@ -191,12 +188,12 @@ this API:
 to trigger attribution for all matching sources.
 
 The browser will treat redirects to a URL of the form:
-`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?data=<data>&priority=<priority>]`
+`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?data=<data>&priority=<priority>&dedup-key=<dedup-key>]`
 
-as a special request, where optional data associated with the event that triggered attribution is stored in a query parameter.
-
-- `data`: an integer label used for the trigger, limited to 3 bits.
-- `priority`: a signed 64-bit integer representing the priority of this trigger compared to other triggers for the same source.
+as a special request. The query string for the request contains additional information about the attribution trigger:
+- `data`: optional data to identify the triggering event
+- `priority`: optional signed 64-bit integer representing the priority of this trigger compared to other triggers for the same source.
+- `dedup-key`: optional signed 64 bit integer which will be used to deduplicate multiple triggers which contain the same dedup-key for a single source
```

Comment:
nit: `64 bit` is spelled inconsistently (line 195 has it as a `64-bit`)

Reply:
Fixed, thanks!